index.ts

```diff
@@ -80,14 +82,23 @@ export class SpotifyClient {
     }
   };
 
-  getLastPlayed = async () => {
+  getLastPlayed = async (limit: number = 1): Promise<ResponseTrack[]> => {
```

Comment:
After this change, this will return an Array instead of an object. This is a breaking change and needs to be updated in the readme example too.